# state machine configurations
FixPoseX : 0  
FixPoseY : 0 
FixPoseZ : 10  
